Test/git-issues/git-issue-4188.dfy

```diff
@@ -0,0 +1,14 @@
+// RUN: %baredafny verify %args "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+newtype Int = nat
```

Comment:
I suggest making two more variations of this example, namely the ones mentioned in the Issue discussion (`newtype Int = int` and `newtype Int = n: nat | true`, perhaps also with `newtype Int = x: int | 0 <= x`).